src/util/script_detection.js

```diff
@@ -306,6 +309,15 @@ export function charInSupportedScript(char: number, canRenderRTL: boolean) {
     return true;
 }
 
+export function stringContainsRTLText(chars: string): boolean {
+    for (const char of chars) {
+        if (charInRTLScript(char.charCodeAt(0))) {
```

Comment:
Is it possible that a string could contain RTL text if the first character isn't in one of the blocks in the `charInRTLScript` function? Do Hebrew and Arabic have anything like the mixture of Arabic numerals and Latin characters with Japanese characters?

Reply:
Re-reading this, I realize I just asked if Arabic mixes in Arabic numerals which....🤦‍♂but it does add to my question of whether or not this is a reliable detection method